### PR TITLE
added order page tests

### PR DESCRIPTION
**Pull request by utkalbarik**
Hi Utkal,

Please review the PR for order page??? 
Added testpage and test cases...

### CODE REVIEW

File: src/pages/OrderPage.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package pages;
+
+public class OrderPage {
+
```
Comment (utkalgituser) [inline src/pages/OrderPage.java]: Where is order page actions/methods? 
where is the logic?

File: src/test/OrderTest.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package test;
+
+public class OrderTest {
+	public void orderTest() {
+		System.out.println(true);
```
Comment (utkalgituser) [inline src/test/OrderTest.java:5]: Variable name is missing??

**Review by utkalgituser — CHANGES_REQUESTED** [review]
Please check my comments and do the required changes. 

Thanks
Utkal

File: src/test/OrderTest.java (hunk 2 above)
Comment (utkalbarik) [reply src/test/OrderTest.java:5]: Assertion added.

File: src/pages/OrderPage.java (hunk 1 above)
Comment (utkalbarik) [reply src/pages/OrderPage.java]: orderId added.

**Review by utkalgituser — APPROVED** [review]
PR is approve.

**Pull request merged**